integ-test/src/test/resources/multi_nested_objects.json

```diff
@@ -1,9 +1,9 @@
 {"index":{"_id":"1"}}
-{"message":[{"info":"a","author":{"name": "e", "address": {"street": "bc", "number": 1}},"dayOfWeek":1}]}
```

Comment:
If we are updating the test data, maybe we could update the values to be better then one or two single letters.